content/news/playtest-20170827.md

```diff
@@ -0,0 +1,19 @@
+A month after the previous playtest, we're pleased to announce the new release candidate, playtest-20170827.
+
+In addition to all the new features and improvements [the previous playtest](http://www.openra.net/news/playtest-20170722/) brought along, in this new playtest we fixed over 30 issues that were identified through feedback from players and modders. Among the more noticable ones were the movement 'stuttering' that ground units were showing, as well as some issues with the new command bar and some other UI hiccups.
+
+Furthermore, the Red Alert mod in particular received a larger balance overhaul, along with a new map (Agenda). 'Agenda' and the updated 'Great Sahara 2' now serve as examples of more prominent use of civilian 'tech' structures that provide various benefits when captured.
+
+<div style="text-align:center" markdown="1">
+![New civilian tech buildings](/images/news/20170827-agenda-civ-tech.png)
+
+The capturable civilian tech structures on the new map Agenda.
```

Comment:
Would be good if this could mention in a couple of words what the two new tech structures do.